ZAPD/ZTextureAnimation.cpp

```diff
@@ -608,13 +569,20 @@ void ZTextureAnimation::DeclareReferences(const std::string& prefix)
 			if (!parent->HasDeclaration(paramsOffset))
 			{
 				ZTextureAnimationParams* params;
-				int count = 2;
+				int count;
 				switch (entry.type)
 				{
 				case TextureAnimationParamsType::SingleScroll:
-					count = 1;
-					[[fallthrough]];
-				case TextureAnimationParamsType::DualScroll:
+					if (true)
+					{
+						count = 1;
+					}
+					else
+					{
+						[[fallthrough]];
+					case TextureAnimationParamsType::DualScroll:
+						count = 2;
+					}
```

Comment:
Seems like there's some unreachable code here. Is this some kind of hack/workaround?

Reply:
The case label allows you to jump into the else. The idea is that in the SingleScroll case, it sets `count = 1` and falls through to the rest of the scroll-setting code while avoiding setting `count` to `2`, while in the DualScroll case, it jumps into the else, sets it to 2, and carries on to the rest of the code. I thought it was less confusing than setting it to 2 and then setting it to 1 only in the SingleScroll case and using a fallthrough, but maybe not?

Reply:
I think it would be a good idea to add a comment explaining how that works.